COCO_DATA_FORMAT.md

```diff
@@ -108,6 +108,44 @@ Specifically, **only** image files are supported for the label files. The ground
 }
 ```
 
+
+## Visual Question Answering
+
+This data kind represents the problem where one asks a question about an image and a grounth  is associated.
+
+```json
+{
+    "images": [
+        {"id": 1, "zip_file": "test1.zip", "file_name": "test/0/image_1.jpg"},
+        {"id": 2, "zip_file": "test2.zip", "file_name": "test/1/image_2.jpg"}
+    ],
+    "annotations": [
+        {"image_id": 1, "id": 1, "question": "what animal is in the image?", "answer": "a cat"},
+        {"image_id": 2, "id": 2, "query": "What is the title of the book on the shelf?", "answer": "How to make bread"}
+    ]
+}
+```
+
+
+## Visual Object Grounding
+
+This is Visual Question & Answering with bboxes in the groundtruth.
+
+
+```json
+{
+    "images": [
+        {"id": 1, "zip_file": "test1.zip", "file_name": "test/0/image_1.jpg"},
+        {"id": 2, "zip_file": "test2.zip", "file_name": "test/1/image_2.jpg"}
+    ],
+    "annotations": [
+        {"image_id": 1, "id": 1, "question": "whats animal are in the image?", "answer": "cat and bird", "grounding": [{"text": "a cat", "bbox": [10, 10, 100, 100]}, {"text": "a bird", "bbox": [15, 15, 30, 30]}]},
+        {"image_id": 2, "id": 2, "question": "What is the title and auther of the book on the shelf?", "answer": "Tile is baking and auther is John", : [{"text": "Title: Baking", "bbox": [10, 10, 100, 100]}, {"text": "Author: John", "bbox": [0, 0, 50, 50]}]}
```

Comment:
can we add an id to the grounding  [{"text": "Title: Baking", "bbox": [10, 10, 100, 100], "id":1}, {"text": "Title: Baking", "bbox": [1, 10, 100, 200], "id":2}